route/build/src/mc_route.f90

```diff
@@ -213,17 +213,12 @@ SUBROUTINE mc_rch(iEns, segIndex, & ! input: index of runoff ensemble to be proc
    write(iulog,'(A,X,G12.5)') ' RCHFLX_out(iens,segIndex)%REACH_Q=', RCHFLX_out(iens,segIndex)%ROUTE(idxMC)%REACH_Q
  endif
 
- if(verbose) then
-   write(iulog,'(a)') ' -------------------------'
```

Comment:
Good to move this block to a shared subroutine.

Reply:
thanks!